### DISCUSSION TITLE
Move verification code upwards in emails

### DISCUSSION THREAD

**Discussion by gnojus** [community/community#5068] [Other Feature Feedback, Questions, & Ideas]
Current verification email looks like this:
```
Hey <user>!

A sign in attempt requires further verification because we did not recognize your device. To complete the sign in, enter the verification code on the unrecognized device.

Device: <device>
Verification code: <code>
...
```
because of the paragraph above the actual code, the code is not visible in my mobile notifications. This means that I need to open the email app. It would me more convenient if the code was placed earlier in text.
e. g:
```
Hey <user>!

Your verification code: <code>

A sign in attempt requires further verification because we did not recognize your device. To complete the sign in, enter the verification code on the unrecognized device.

Device: <device>
...
```
Thanks

**Comment by spacewalkingninja** [comment]
We don't even wanna verify! We just WANT NO CODES AND NO 2FA. ONLY PASSWORD. 
Yeah i also hate that paswordless idea strange thing whatever. I love my passwords.
